x-pack/plugins/security_solution/public/timelines/components/timeline/body/actions/index.tsx

```diff
@@ -201,6 +205,25 @@ const ActionsComponent: React.FC<ActionProps> = ({
     scopedActions,
   ]);
 
+  const { isTourShown, incrementStep } = useTourContext();
+
+  const isTourAnchor = useMemo(
+    () =>
+      isTourShown(SecurityStepId.alertsCases) &&
+      eventType === 'signal' &&
+      // TODO: Steph make sure this is right
```

Comment:
I think it is. I thought most of the shared components had `timelineId` switched to `scopeId`, but I think the actions were a bit different.

Reply:
will remove this comment from code in a follow up PR, close to that sweet sweet green build